packages/react-scripts/template/README.md

```diff
@@ -346,6 +346,35 @@ Learn more about ES6 modules:
 * [Exploring ES6: Modules](http://exploringjs.com/es6/ch_modules.html)
 * [Understanding ES6: Modules](https://leanpub.com/understandinges6/read#leanpub-auto-encapsulating-code-with-modules)
 
+## Importing a Component using path relative to src (Absolute path)
+
+Create a file called `.env` in the root of the project, then add the following:
```

Comment:
Yeah, but i believe that .env should contain the environment specific setting like `CHOKIDAR_USE_POLLING` which doesn't need to turned on in specific env. This env variable is making the codes in `src` directly dependent to have it setup.

So, this would cause confusion if the docs says it like this right now. When the original author push it to a git repo and other team member tries to clone it, the repo is not working in other team member local machine because of the missing .env files. And we have "it's working on my local machine" problem all over again.

I think right now If we want to document `NODE_PATH` in the docs we need to use the cross-env in the scripts, or make it configurable. 

The officially supported ways of making an absolute import is discussed here https://github.com/facebookincubator/create-react-app/issues/1065 so I think that this pr would be better to document that instead.